app/_locales/en/messages.json

```diff
@@ -2175,6 +2201,34 @@
   "permissionRequest": {
     "message": "Permission request"
   },
+  "permission_accessNetwork": {
+    "message": "Access the Internet.",
+    "description": "The description of the `endowment:network-access` permission."
+  },
+  "permission_accessSnap": {
+    "message": "Connect to the $1 Snap.",
+    "description": "The description for the `wallet_snap_*` permission. $1 is the name of the Snap."
+  },
+  "permission_customConfirmation": {
+    "message": "Display a confirmation in MetaMask.",
+    "description": "The description for the `snap_confirm` permission"
+  },
+  "permission_ethereumAccounts": {
+    "message": "See address, account balance, activity and initiate transactions",
+    "description": "The description for the `eth_accounts` permission"
+  },
+  "permission_manageBip44Keys": {
+    "message": "Manage your \"$1\" keys.",
```

Comment:
Why was this text used rather than the one in the designs? That said "grant irrevocable control". This description does not convey the risks of granting this permission.

Reply:
This has been updated.